esp-hal/src/soc/esp32s3/gpio.rs

```diff
@@ -74,6 +74,7 @@ macro_rules! rtcio_analog {
                 $crate::ignore!($analog);
                 impl $crate::peripherals::[<GPIO $pin_num>]<'_> {
                     /// Configures the pin for analog mode.
+                    #[allow(unused)]
```

Comment:
No idea what's going on here, you can see the previous failure without this attribute [here](https://github.com/jessebraham/esp-hal/actions/runs/17760147126/job/50470858056#step:7:325). Probably I'm missing something, but couldn't figure out why this just cropped up.